site/src/pages/components/menu.explainer.mdx

```diff
@@ -26,101 +26,111 @@ layout: ../../layouts/ComponentLayout.astro
+  referenced by its ID from its invoker.
+* Provides the correct keyboard and focus behavior across sub-menulists, menubars, and child menu
+  items.
 
-* Menulist should be used to represent submenus of commands.
-* In a menulist, a menu item's most common purpose is to invoke actions (commands) or other submenus (menulists).
-* Menulist is a **popover** menu that can contain a list of `<menuitem>`s.
-* Anchored to the button which is declaratively set up to open it.
-* Attributes
-    * disabled, open, name?
+**Content model**: the `<menulist>` element allows the following elements as descendants:
 
-**Question: Is `<fieldset>` necessary to group `<menuitem>` together?**
+* `<menuitem>`
+* `<hr>`
+* `<fieldset>`
+* **Interactive content**: When interactive content appears in a menu, it will be a sibling of
+  `<menuitem>`s, not nested inside them, as nested interactive content is generally inaccessible.
+  See [Issue #1323](https://github.com/openui/open-ui/issues/1323) for more discussion on how we
+  might support other interactive content in menus, such as `<input type=search>`.
+* Anchor `<a>` tags are disallowed in menu lists, due to accessibility concerns around exposing
+  links in ARIA menus, and concerns around the navigation menubar pattern. We enforce this with the
+  content model, and propose default `display: none` styles for anchors that are direct descendants
+  of `<menulist>` elements.
 
-Yes. While, we can use the <code>["name"](https://html.spec.whatwg.org/C#attr-fe-name)</code>
-attribute to logically group exclusive checkable `<menuitem>`s together, to support captioned groups
-of `<menulist>` items, we should use the existing `<fieldset>` element (and its optional labeling
-`<legend>` element) that give this functionality to forms. If multiple `<menuitem>`s are added
-without a `<fieldset>`, they are grouped together under their parent `<menulist>` or `<menubar>`,
-but are logically separate and are not checkable.
+The `<fieldset>` element is used in a `<menulist>` to group menu items that serve a common purpose.
+It gives two key pieces of functionality:
 
-Note, `<fieldset>` is a more general grouping element than `<optgroup>`, which we briefly
-considered, but decided against since it is used for grouping options.
+* Captioned groups, with its optional `<legend>` for labeling.
+* A common ancestor for checkable `<menuitem>`s. This supports both exclusively checkable and
+  multi-checkable menu items.
 
-Our proposal includes adding a `checkable` attribute on `<fieldset>`, to group `<menuitem>`s as
-either radios or checkboxes, instead of introducing new `<menuitemcheckbox>` and `<menuitemradio>`
-elements. We envision the following markup:
+**Checkable menu items**
+
+To support checkable menu items, we propose the new `checkable` **content** attribute on
+`<fieldset>` (no corresponding **IDL** attribute), to group items as either radios or checkboxes;
```

Comment:
nit: HTML will require you to add an IDL reflection. Any reason not to provide one?

Reply:
Hmm, I'm not sure why I had it this way originally, but yeah I see no special reason to avoid reflection.